src/components/ReleaseMonitor/ReleaseListHeader.tsx

```diff
@@ -50,4 +53,37 @@ const releaseColumns = [
   },
 ] satisfies Parameters<typeof createTableHeaders>[0];
 
+export const getReleasesListHeader = (
+  activeSortIndex: number,
+  activeSortDirection: SortByDirection,
+  onSort: ThProps['sort']['onSort'],
+  totalCount?: number,
+) => {
+  return (componentProps: ComponentProps) => {
+    const baseHeaders = createTableHeaders(releaseColumns)(
+      activeSortIndex,
+      activeSortDirection,
+      onSort,
+    )(componentProps);
+
+    if (totalCount !== undefined) {
+      const countColumn = {
+        title: (
+          <span style={{ fontWeight: 'bold', fontSize: '0.875rem' }}>
+            {totalCount} {totalCount === 1 ? 'release' : 'releases'}
```

Comment:
nit: you can use `pluralize` util func from `@patternfly/react-core` lib. We use it in a couple of places in the codebase, so you'll be able to find examples to inspire from :)